### ISSUE TITLE
Questions around missing functionality 

### ISSUE THREAD

**Issue by roberthutto** [jenkinsci/fortify-plugin#45]
I recently tried to use this plugin within my organization that uses SSC. In our workflow, we do a local translate and then a remote scan. We also download the .fpr to Jenkins. I'm using the "-block -f <file_name>.fpr" to cause the remote scan to block, wait for status complete, and download the .fpr. I don't see how to accomplish this with fortifyRemoteScan. Maybe I missing something. If it's not existing functionality it does not seem like much work to add. 

How often do you release? Is this something I can contribute?

@young-s-park @akaryakina

**Comment by akaryakina** [comment]
I don't think that you can do it with the plugin right now. But this feature is on our radar and will probably make it in the next release or so. Yeah, you're definitely welcome to contribute, just make a PR and we'll discuss further.
We don't have a release schedule and it depends on our resource availability and market/customer demand. You can look at our release list to see the history. It usually happens either once or twice per year.
